unfetter-discover/unfetter-ui
# Issue 1246

Refactor of the assessment summary page with new failed to load text.

To test:
Go to assessments beta
Load several different assessments and check that the assessment summary page loads as expected
Change the assessment id in the url to a bogus one (add a character or two)
Check that text about unable to find this assessment displays
